From ffeb0072bbd1d1738a685cbfcc12776ac9c63a82 Mon Sep 17 00:00:00 2001
From: Pradeep Sriram <pradeep.sriram@amlogic.com>
Date: Tue, 5 Nov 2019 22:20:46 -0800
Subject: [PATCH] RDK: migrate bootloader code base to trunk [1/3]

PD#SWPL-24384

Problem:
  gcc path is hardcoded

Solution:
  Remove hardcoded path

Verify:
  Local

Signed-off-by: Pradeep Sriram <pradeep.sriram@amlogic.com>
Change-Id: Idfe7c3052ad5d4c46f82b233f95ae9062f684c06
---
 Makefile                                      | 2 +-
 arch/arm/mach-meson/sc2/firmware/acs/Makefile | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index e94ba6f1d0..40d00b47c7 100644
--- a/Makefile
+++ b/Makefile
@@ -249,7 +249,7 @@ ifeq ($(HOSTARCH),$(ARCH))
 CROSS_COMPILE ?=
 endif
 
-CROSS_COMPILE ?=/opt/gcc-linaro-7.3.1-2018.05-i686_aarch64-elf/bin/aarch64-elf-
+CROSS_COMPILE ?=aarch64-elf-
 export CROSS_COMPILE
 
 KCONFIG_CONFIG	?= .config
diff --git a/arch/arm/mach-meson/sc2/firmware/acs/Makefile b/arch/arm/mach-meson/sc2/firmware/acs/Makefile
index 01d869a755..0245fa71a1 100644
--- a/arch/arm/mach-meson/sc2/firmware/acs/Makefile
+++ b/arch/arm/mach-meson/sc2/firmware/acs/Makefile
@@ -32,7 +32,7 @@
 # Default values for build configurations
 #
 
-export CROSS_COMPILE=/opt/gcc-linaro-aarch64-none-elf-4.8-2013.11_linux/bin/aarch64-none-elf-
+export CROSS_COMPILE=aarch64-none-elf-
 
 include $(buildsrc)/config.mk
 include $(buildtree)/include/autoconf.mk
-- 
2.17.1

